projects/ipdg_stokes/post_processing/mesh_function_interpolation.h

```diff
@@ -0,0 +1,105 @@
+#ifndef THESIS_POST_PROCESSING_MESH_FUNCTION_INTERPOLATION_H
+#define THESIS_POST_PROCESSING_MESH_FUNCTION_INTERPOLATION_H
+
+/**
+ * @file mesh_function_interpolation.h
+ * @brief Bring mesh functions down in a mesh hierarchy
+ */
+
+#include <lf/base/lf_exception.h>
+#include <lf/refinement/mesh_hierarchy.h>
+
+#include <vector>
+
+namespace projects::ipdg_stokes {
+
+namespace post_processing {
+
+/**
+ * @brief A MeshFunction representing another MeshFunction on a coarser mesh
+ * @tparam INNER_MF The MeshFunction on the coarser mesh
+ */
+template <typename INNER_MF>
+class MeshFunctionInterpolation {
```

Comment:
I guess this will be replace by MeshFunctionTransfer eventually?

Reply:
Yes. However, that is currently still in development. As soon as it is finished though, I will replace this with the LehrFEM++ internal routines.

Reply:
Great :)